src/pybind/rados.py

```diff
@@ -1984,6 +1984,23 @@ def lookup_snap(self, snap_name):
             raise make_ex(ret, "Failed to lookup snap %s" % snap_name)
         return Snap(self, snap_name, snap_id)
 
+    @requires(('oid', str_type), ('snap', Snap))
+    def snap_rollback(self, oid, snap):
```

Comment:
this would be more consistent with the other snap methods if it took the value needed, i.e. the snap_name, rather than a Snap object